include/boost/url/detail/impl/remove_dot_segments.ipp

```diff
@@ -187,8 +187,8 @@ int
 normalized_path_compare(
     string_view s0_init,
     string_view s1_init,
-    bool s0_remove_unmatched,
-    bool s1_remove_unmatched) noexcept
+    bool r0,
+    bool r1) noexcept
```

Comment:
it would still be nice to know what these mean

Reply:
The declaration uses `remove_unmatched_lhs` and the definition uses `r0`. I've been changing this back and forth. Any other name recommendation? Or any recommendation of a place to include a comment?